rabbitmq-operator/src/main/java/com/indeed/operators/rabbitmq/reconciliation/rabbitmq/ShovelReconciler.java

```diff
@@ -54,7 +55,10 @@ public void reconcile(final RabbitMQCluster cluster) {
                     .map(addr -> String.format("amqp://%s:%s@%s", username, password, addr.asRabbitUri()))
                     .collect(Collectors.toList());
 
-            final ShovelArguments shovelArguments = new ShovelArguments().withSrcQueue(desiredShovel.getSource().getQueue()).withDestUri(uris);
+            final ShovelArguments shovelArguments = new ShovelArguments()
+                    .withSrcUri(Lists.newArrayList("amqp://"))
```

Comment:
`amqp://` seems like an odd source URI.  Is that correct?

Reply:
yep, it just means "this cluster". moved into a const.